README.md

```diff
@@ -101,6 +101,10 @@ The `scripts` folder in this repository contains scripts that interact with the
 
   Checks all free-text fields in G-Cloud services for "bad words" and generates a CSV report of any bad words found.
 
+* `send_dos_opportunities_email.py`
```

Comment:
@samuelhwilliams you are totally right that this name is not consonant with the rest, but if we change it, we also need to change the Jenkins job and this probably is outside of scope of this manual update. Should we add it to the tech debt?